lib/dap/filter/quic.rb

```diff
@@ -0,0 +1,38 @@
+module Dap
+   module Filter
+
+      #
+      # Decode an Quic VersionsRequest probe response
+      #
+      class FilterDecodeQuicVersionsResult
+        include BaseDecoder
+
+        #
+        # Decode an QUIC versions probe response
+        #
+        # @param data [String] Binary string containing raw response from server
+        # @return [List] Hash containing all LDAP responses
```

Comment:
I believe this should return `[Hash]` and appears to return version info? or nil if no versions found.